### PR TITLE
Add support for individual repos

### PR DESCRIPTION
**Pull request by monfresh**
**Why**: Some teams might want to integrate Seal into their Slack
channel for their individual repo. This PR allows individual repos to
be added to the YAML config.

### CODE REVIEW

**Comment by monfresh** [comment]
cc @jessieay @ccostino. Could I trouble you for another review? This can wait until Monday.

File: lib/github_fetcher.rb [hunk 1]
```diff
@@ -51,6 +51,10 @@ def language
     team_config['language']
   end
 
+  def repo
+    "#{ORGANISATION}/#{team_config['repo']}"
```
Comment (jessieay) [inline lib/github_fetcher.rb:55]: what's the deal with the british spelling? (s instead of z) ?

File: README.md [hunk 2]
```diff
@@ -38,12 +62,15 @@ ignored_repos:
...
 ```
 
-Configure the required environment variables in your shell profile:
+### Environment variables
+
+Configure the required environment variables. To test the bot locally, you
+can set the variables by running the following commands in your Terminal:
 
 ```sh
 export SEAL_ORGANISATION="your_github_organisation"
```
Comment (ccostino) [inline README.md:71]: Same question as @jessieay, for consistency's sake. :-)

File: spec/github_fetcher_spec.rb [hunk 3]
```diff
@@ -60,8 +60,8 @@
 
   let(:team_config) do
     {
-      'language' => nil,
-      'channel' => nil
+      'language' => 'ruby',
+      'channel' => '#ruby'
```
Comment (ccostino) [inline spec/github_fetcher_spec.rb:64]: Should you also test for the case of an improperly configured YAML file, e.g., one with no languages set?  What happens in that situation, anyway?  Should the bot raise a warning that you have nothing set and therefore it won't actually do anything?

This doesn't have to be considered for this PR, just a thought I had triggered by this change.

File: lib/github_fetcher.rb (hunk 1 above)
Comment (monfresh) [inline lib/github_fetcher.rb:55]: The original project ours was forked from was built by a British person :) We can certainly change it, but I would do that in a separate PR since the naming isn't new to this PR.

File: spec/github_fetcher_spec.rb (hunk 3 above)
Comment (monfresh) [inline spec/github_fetcher_spec.rb:64]: Good point. I'll address that in a separate PR.

**Comment by ccostino** [comment]
Ah good note about the fork for the language questions we had. :-)  The changes look good to me otherwise, @monfresh!
